README.md

```diff
@@ -1,3 +1,8 @@
+## Requirements
+- AWS CLI [download link](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- nodejs [download link](https://nodejs.org/en/download/)
+- Docker [download link](https://docs.docker.com/get-docker/)
+
```

Comment:
This should go below the main heading and description of the project. (Right after SDS-data-manager and before AWS Setup) Users want to know what the project is before the requirements we need for the project.